### PR TITLE
Added information about migration to 2.x

### PR DESCRIPTION
**Pull request by zerc**
Давайте посмотрим, мне кажется не очень ясно получилось написать, может какие идеи появятся в процессе.

---
This change is [Reviewable](https://reviewable.io/reviews/uploadcare/pyuploadcare/110)

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -39,3 +39,26 @@ After that you can install it:
...
+
+Update to version 2.0
+---------------------
+
+Some caveats about migration process from version 1.x to 2.x.
+
+A version 2.0 contains the next breaking changes:
+
+Changed initializing for the ``FileList`` and ``GroupList`` classes.
+The ``since`` and ``until`` parameters has been removed. Use combination from
```
Comment (dmitry-mukhin) [inline docs/install.rst]: have been
combination of

File: docs/install.rst [hunk 2]
```diff
@@ -39,3 +39,26 @@ After that you can install it:
...
+``starting_point`` and ``ordering`` instead.
+
+The ``ucare list`` CLI command has been renamed to ``ucare list_files``. And
```
Comment (dmitry-mukhin) [inline docs/install.rst]: And,

File: docs/install.rst [hunk 3]
```diff
@@ -39,3 +39,26 @@ After that you can install it:
...
+according to the previous change, the ``since`` and ``until`` parameters has
+been removed here. The ``starting_point`` and ordering parameters added.
```
Comment (dmitry-mukhin) [inline docs/install.rst]: ~~here~~

File: docs/install.rst [hunk 4]
```diff
@@ -39,3 +39,26 @@ After that you can install it:
...
+
+These changes a necessary for working with 0.5 version of REST API.
```
Comment (dmitry-mukhin) [inline docs/install.rst]: are
of the REST API

Comment (dmitry-mukhin) [inline docs/install.rst]: version 0.5

File: docs/install.rst [hunk 5]
```diff
@@ -39,3 +39,26 @@ After that you can install it:
...
+So that means you can’t use these classes correctly with versions prior 0.5.
```
Comment (dmitry-mukhin) [inline docs/install.rst]: prior 0.5 (but that should not be an issue :).

File: docs/install.rst [hunk 6]
```diff
@@ -39,3 +39,26 @@ After that you can install it:
...
+
+Also, note that setting ``UPLOADCARE['widget_variant']`` now is deprecated and
```
Comment (dmitry-mukhin) [inline docs/install.rst]: Django configuration option?

File: docs/install.rst [hunk 7]
```diff
@@ -39,3 +39,26 @@ After that you can install it:
...
     $ python setup.py install
+
+
+Update to version 2.0
+---------------------
+
+Some caveats about migration process from version 1.x to 2.x.
+
+A version 2.0 contains the next breaking changes:
+
```
Comment (dmitry-mukhin) [inline docs/install.rst]: is this a list or what?

Comment (zerc) [inline docs/install.rst]: Turn next two paragraphs into a list.

**Comment by zerc** [comment]
bump

File: docs/install.rst [hunk 8]
```diff
@@ -48,17 +48,14 @@ Some caveats about migration process from version 1.x to 2.x.
...
-So that means you can’t use these classes correctly with versions prior 0.5.
+These changes are necessary for working with version 0.5 of REST API.
+So that means you can’t use these classes correctly with versions prior 0.5
+(but that should not be an issue :)
 
-Also, note that setting ``UPLOADCARE['widget_variant']`` now is deprecated and
-it’s will be removed in next major release. Use ``UPLOADCARE['widget_build']``
-instead.
+Also, note that Django configuration option ``UPLOADCARE['widget_variant']``
+now is deprecated and it’s will be removed in next major release. Use
```
Comment (dmitry-mukhin) [inline docs/install.rst]: it will be

Comment (dmitry-mukhin) [inline docs/install.rst]: otherwise shipit

**Pull request merged**